### PR TITLE
Add definition of done, estimated effort, and descriptive title notes

### PR DESCRIPTION
**Pull request by jverbraeken**
- Relevant Issues: -
- Related Pull Requests: -

* * *

## What
This PR improves the clarity of issues

## Why
To make the task more clear for the ones assigned to the tasks

## How
The PR adds the definition of done, the estimated effort and a comment that asks you to check if the title is descriptive enough to the issue template.

## Notes
-

### CODE REVIEW

**Comment by ericcornelissen** [comment]
Can you fill out the PR template :innocent:

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,19 +1,24 @@
-- Parent issues: ...
-- Relevant issues: ...
+[//]: # "Check if the title is descriptive!"
+- Parent issues: -
+- Relevant issues: -
+- Estimated effort: PLACEHOLDER
```
Comment (ericcornelissen) [inline ISSUE_TEMPLATE.md]: Can you just use a `-` here for consistency sake 😅

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
 ## Why
-[Optional] Specify in more detail why this issue is needed in the project.
+Specify in more detail why this issue is needed in the project.
 
 ## How
-[Optional] Specify, in general terms, how this feature can be added.
+Specify in general terms how this feature can be added.
+
+## Definition of done
+Specify what must be done exactly before this task is done.
```
Comment (ericcornelissen) [inline ISSUE_TEMPLATE.md]: Not the clearest sentence imo, maybe: "Specify exactly what must be done before this task can be considered done"

**Comment by jverbraeken** [comment]
@ericcornelissen Done!

**Pull request merged**